### PR TITLE
Create strings.xml

### PR DESCRIPTION
**Pull request by netnotix**
add german translation

### CODE REVIEW

File: com.koushikdutta.inkwire/res/values-de/strings.xml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<resources>
+    <string name="app_name">Inkwire</string>
+    <string name="welcome_to_inkwire">Willkommen bei Inkwire</string>
+    <string name="screenshare_in_progress">Dein Bildschirm wird übertragen.</string>
+    <string name="share_this_android">Übertrage deinen Bildschirm an einen anderen Nutzer</string>
+    <string name="share">Freigeben</string>
+    <string name="share_access_code">Zugangscode weitergeben</string>
+    <string name="see_shared_android">Betrachte den Bildschirm eines anderen Nutzers</string>
+    <string name="access">Verbinden</string>
+    <string name="share_access_code_info">Um die Übertragung zu beginnen sende den folgenden Zugangscode an die Person, die Dir helfen soll.</string>
```
Comment (mjk912) [inline com.koushikdutta.inkwire/res/values-de/strings.xml]: Add a missing comma between "beginnen" and "senden"

File: com.koushikdutta.inkwire/res/values-de/strings.xml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    <string name="creating_access_code">Zugangscode erzeugen</string>
+    <string name="send_link">Link senden</string>
+    <string name="copy_link">Link kopieren</string>
+    <string name="or_send_link">Oder sende einen Zugangslink:</string>
+    <string name="or_invite_link">Oder sende einen Einladungslink mit der Bitte den Bildschirm zu teilen:</string>
```
Comment (mjk912) [inline com.koushikdutta.inkwire/res/values-de/strings.xml]: Add a missing comma after "Einladungslink".

File: com.koushikdutta.inkwire/res/values-de/strings.xml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    <string name="waiting_for_connection">Warte auf die Verbindung...</string>
+    <string name="link_copied">Link in die Zwischenablage kopiert</string>
+    <string name="allow_screen_capture">Screenshot erlauben</string>
+    <string name="remote_user_info">Der andere Nutzer ist in der Lage deinen Bilschirm zu sehen und darauf zu zeichnen. Du kannst die Verbindung jederzeit in der Benachrichtigungsleiste beenden.</string>
```
Comment (mjk912) [inline com.koushikdutta.inkwire/res/values-de/strings.xml:19]: Add a missing comma after "Bildschirm"

File: com.koushikdutta.inkwire/res/values-de/strings.xml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    <string name="got_it">gefunden</string>
+    <string name="enable_the_microphone">Aktiviere das Mikrofon um die Sprachverbindung zu starten.</string>
```
Comment (mjk912) [inline com.koushikdutta.inkwire/res/values-de/strings.xml]: Add a missing comma after "Mikrofon"

File: com.koushikdutta.inkwire/res/values-de/strings.xml [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    <string name="error_contacting_server">Es gab einen Fehler beim Verbinden zu den Inkwire Servern.</string>
+    <string name="connecting_dotdotdot">verbinde&#8230;</string>
+    <string name="you_were_invited">Ein anderer Inkwire Nutzer möchte seinen Bildschirm mit dir teilen.</string>
+    <string name="connect">Verbinden</string>
+    <string name="cancel">Abbrechen</string>
+    <string name="session_not_found">Inkwire Sitzung nicht gefunden. Diese ist möglicherweise abgelaufen.</string>
+    <string name="connected_to_screenshare">Du siehst einen geteilten Bildschirm.</string>
+    <string name="enable_microphone">Mikrofon ein</string>
+    <string name="no_thanks">Nein. Danke</string>
+    <string name="you_were_requested">Ein anderer Inkwire Nutzer bittet Dich deinen Bildschirm mit ihm zu teilen.</string>
```
Comment (mjk912) [inline com.koushikdutta.inkwire/res/values-de/strings.xml]: Add a missing comma after "bittet Dich"

Comment (netnotix) [reply com.koushikdutta.inkwire/res/values-de/strings.xml]: Thank you ;-)

File: com.koushikdutta.inkwire/res/values-de/strings.xml [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+    <string name="enter_access_code_info">Bitte den Nutzer, auf dessen Android Du zugreifen möchtest, auf "Freigeben" zu klicken und dir den Zugangscode zu geben.</string>
+    <string name="access_code">Zugangscode</string>
+    <string name="creating_invite_link">Einladungslink erzeugen</string>
+    <string name="error_contacting_server">Es gab einen Fehler beim Verbinden zu den Inkwire Servern.</string>
+    <string name="connecting_dotdotdot">verbinde&#8230;</string>
+    <string name="you_were_invited">Ein anderer Inkwire Nutzer möchte seinen Bildschirm mit dir teilen.</string>
+    <string name="connect">Verbinden</string>
+    <string name="cancel">Abbrechen</string>
+    <string name="session_not_found">Inkwire Sitzung nicht gefunden. Diese ist möglicherweise abgelaufen.</string>
+    <string name="connected_to_screenshare">Du siehst einen geteilten Bildschirm.</string>
```
Comment (mjk912) [inline com.koushikdutta.inkwire/res/values-de/strings.xml]: Sounds like: "You are viewing a split screen." 
Maybe something like "Du siehst einen mit Dir geteilten Bildschirm."? 
IDK, a matter of taste. 
And sorry for the commas. ;)

**Pull request merged**